### PR TITLE
changed description

### PR DESCRIPTION
**Pull request by wadie**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,12 +4,12 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-	  <meta name="description" content="Ember Test Selectors - enabling better element selectors in Ember.js tests"/>
-  	<meta name="keywords" content="emberjs, javascript, tests, selectors, data-test" />
+	  <meta name="description" content="qunit-dom High Level DOM Assertions for QUnit"/>
```
Comment (marcoow) [inline index.html]: I think this should be _"qunit-dom - high Level DOM Assertions for QUnit"_

**Comment by wadie** [comment]
This PR includes all the required changes.

**Comment by marcoow** [comment]
But this also includes the commits that remove the `.travis.yml` and the CSS changes for the link colors. You'll want to rebase this on the target branch (`gh-pages`) and only keep the commit that changes the description and title:

```
git fetch
git rebase -i origin/gh-pages
```

See [this](https://help.github.com/articles/about-git-rebase/) for a better explanation of how rebasing works.

**Comment by wadie** [comment]
Can't remove the CSS changes since they're in the same commit with the description change..

**Pull request merged**